tools/terminal/readme.md

```diff
@@ -1,7 +1,9 @@
 # rs-terminal Tool
 
 ## Goal
-`rs-terminal` tool can be used to send debug commands to camera firmware. The tool can operate in two modes - if you have access to the specification of debug commands for the exact firmware you are using (provided in XML form) the tool will provide help and auto-completion based on the information within the XML. If you don't have access to the spec, you can still operate the tool in HEX mode, sending binary commands directly. Later mode of operation is intended to allow Intel support team to troubleshoot the hardware remotely by providing the user the exact binary commands to input. 
+`rs-terminal` tool can be used to send debug commands to camera firmware. The tool can operate in two modes - if you have access to the specification of debug commands for the exact firmware you are using (provided in XML form) the tool will provide help and auto-completion based on the information within the XML.  
+The tool may accept specific parameters input from files as well as the console
+If you don't have access to the spec, you can still operate the tool in HEX mode, sending binary commands directly. Later mode of operation is intended to allow Intel support team to troubleshoot the hardware remotely by providing the user the exact binary commands to input. 
```

Comment:
`Later` -> `This`

This line is meant to follow line 4. Line 5 is out of order -- I suggest moving it after this one (and end it with a period) or removing it.

Reply:
Code reverted as the extra explanation is not essential